test/node-test/diagnostics-channel/fetch.js

```diff
@@ -0,0 +1,124 @@
+'use strict'
+
+const { tspl } = require('@matteo.collina/tspl')
+const { describe, test, before, after } = require('node:test')
+const { fetch } = require('../../..')
+
+let diagnosticsChannel
+let skip = false
+try {
+  diagnosticsChannel = require('node:diagnostics_channel')
+} catch {
+  skip = true
+}
+
+const { createServer } = require('http')
+
+describe('diagnosticsChannel for fetch', { skip }, () => {
```

Comment:
because of #2761 

```suggestion
const { fetch } = require('../../..')
const diagnosticsChannel = require('node:diagnostics_channel')
const { createServer } = require('http')

describe('diagnosticsChannel for fetch', () => {
```